cfgov/jinja2/v1/_includes/molecules/global-search.html

```diff
@@ -14,10 +14,16 @@
 
 {% macro render( language='en' ) %}
 
+{%- set search_url_base = "https://search.consumerfinance.gov/search" -%}
+{%- set search_url = (
+    search_url_base ~ "?utf8=%E2%9C%93&affiliate=" ~ search_gov_affiliate() ~ "&query="
+    "query="
```

Comment:
Not following the purpose of the extra "query=", but it doesn't show up in use.

Reply:
Nice catch! That looks like a rebase/editing error on my part, that actually does break things when JavaScript is disabled:

![image](https://user-images.githubusercontent.com/654645/145108378-5ad82a6a-1875-4fa7-8184-40edf4476131.png)

Will correct.

Reply:
Also was wondering about the stand-in "?utf8=✓" query-string starter. It's often used so all subsequent query bits can start with "?" – but I though specifying the unicode ✓ character forced some older browsers to use UTF8. Wouldn't we use ✓ instead of the url-quoted entity for such a purpose?

Reply:
> It's often used so all subsequent query bits can start with "?"

Can you explain more? This code existed before this PR, and I wasn't really sure the merits of the different options. In the Search.gov docs they also use `value="&#x2713;"` in the form whereas we use `value="✓"`. 

It looks like @anselmbradford added the `utf8=%E2%9C%93` [a mere 16 months ago](https://github.com/cfpb/consumerfinance.gov/commit/e894384dc7bb64526f40417a500c2da4a1fce0e4) - @anselmbradford is there a particular reason why the link is formatted this way?

Reply:
I guess if we're using a dummy query starter, I'd use `?utf=✓` and have it do some good in the world.

https://stackoverflow.com/questions/27338154/why-do-some-websites-have-utf8-in-their-title#answers-header

Reply:
> It's often used so all subsequent query bits can start with ~~"?"~~ "&"

My typo. 

When query-building, you don't have to decide which query term begins with "?" if you always start with `?utf8=✓`

Reply:
Thanks @higs4281; I've added that change and rebased it in to clean up the commits.

Reply:
Yeah I'm not recalling anything on the URL-encoding of the checkmark. This is for the link when the JS is turned off and the browser appears to parse the encoding before it has been clicked. This is hovering the link:
<img width="1025" alt="Screen Shot 2021-12-08 at 9 44 37 AM" src="https://user-images.githubusercontent.com/704760/145228475-b58d5ec0-8f9c-47dc-b11f-2a131474018d.png">